src/controllers/http-api/v1/request-schema/ask-schema-v1.js

```diff
@@ -0,0 +1,15 @@
+export default () => ({
+    type: 'object',
+    required: ['ual', 'blockchain'],
```

Comment:
if `minimumNumberOfNodeReplications` is not required we need to set it somewhere to something